robot/Cumulus/tests/browser/donations_payments/create_donation_and_payment.robot

```diff
@@ -44,17 +47,19 @@ Create Donation from a Contact
 
     Enter Text Field Value               ${date}
     Enter Payment Schedule               ${No_of_payments}      ${intervel}    ${frequency}
-    scroll button into view and click using js       Calculate Payments
+    scroll button into view and click using js                  Calculate Payments
     ${value}                             Verify Payment Split   1000           ${No_of_payments}
     Should be equal as strings           ${value}    ${No_of_payments}
```

Comment:
Can this not be checked in the `Verify Payment Split` functions itself?

Reply:
Did not focus on that as was fixing the intermittent failure. But good one and addressed it.